synthetics-stats/schema.graphql

```diff
@@ -190,26 +190,43 @@ type SwapFeesInfo @entity(immutable: true) {
   transaction: Transaction!
 }
 
-type PoolValue @entity {
-  # marketAddress
-  id: ID!
-  poolValue: BigInt!
-  pendingFeeUsds: [BigInt!]!
-  pendingCollectedMarketFeesInfoIds: [String!]!
-}
-
 type CollectedMarketFeesInfo @entity {
   # marketAddress:period:timestampGroup
   id: ID!
   period: String!
   marketAddress: String!
   feeUsdForPool: BigInt!
+  cummulativeFeeUsdForPool: BigInt!
   feeUsdPerPoolValue: BigInt!
   cumulativeFeeUsdPerPoolValue: BigInt!
-  cummulativeFeeUsdForPool: BigInt!
+  feeUsdPerGmToken: BigInt!
+  cumulativeFeeUsdPerGmToken: BigInt!
+  prevCumulativeFeeUsdPerGmToken: BigInt!
   timestampGroup: Int!
 }
 
+type UserGmTokensBalanceChange @entity(immutable: true) {
+  # account:marketAddress:timestamp
+  id: ID!
+  account: String!
+  marketAddress: String!
+  tokensBalance: BigInt!
+  timestamp: Int!
+  cumulativeIncome: BigInt!
+  cumulativeFeeUsdPerGmToken: BigInt!
+  actualCumulativeFeeUsdPerGmToken: BigInt!
+  index: BigInt!
```

Comment:
could you explain how `index` is used?

Reply:
order matters, I use it for sorting on the frontend side